aiida/cmdline/params/options/main.py

```diff
@@ -145,6 +146,7 @@ def set_log_level(_ctx, _param, value):
     'profile',
     type=types.ProfileParamType(),
     default=defaults.get_default_profile,
+    cls=CallableDefaultOption,
```

Comment:
Is this really the only option that has an expensive callable default?

Reply:
It's the only place where the config/profile is loaded, all the other use the InteractiveOption where this is already handled. But you are right that there likely are other expensive defaults, but I plan to look into this in a followup PR where I will also look at the timings more closely.